step-templates/octopus-redeploy-previous-release.json

```diff
@@ -0,0 +1,75 @@
+  },
+  "Parameters": [
+    {
+      "Id": "9f8d7b00-417c-406e-8788-326fd67d22b3",
+      "Name": "redeploy.octopus.server.uri",
+      "Label": "Octopus URI",
+      "HelpText": "Provide the URI to your Octopus server.",
+      "DefaultValue": "#{Octopus.Web.ServerUri}",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "7b2ed439-b215-42fe-8971-3839626988fd",
+      "Name": "redeploy.api.key",
+      "Label": "API Key",
+      "HelpText": "Provide an API key with permission to re-deploy the previous version.",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Sensitive"
+      }
+    },
+    {
+      "Id": "92461617-cab6-4dc7-8fdd-0fcfc08c5d66",
+      "Name": "redeploy.release.number",
+      "Label": "Release number",
+      "HelpText": "(Optional)\nProvide a release number to re-deploy.  Leave blank to use the immediate previous release number.",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "47eaaeaf-e414-4b34-ba7d-7d84715fd2ea",
+      "Name": "redeploy.prompted.variables",
+      "Label": "Prompted variables",
+      "HelpText": "Enter any prompted variables and their values in the format `VariableName::VariableValue`.  Enter one entry per line.\n\nExample: `MyVar::MyValue`",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "MultiLineText"
+      }
+    },
+    {
+      "Id": "c0137294-b4e3-406e-a36d-fd7aca2731e7",
+      "Name": "redeploy.prompted.useexisting",
+      "Label": "Use previous prompted variables",
+      "HelpText": "Tick this box to use the same prompted variables and values from the previous deployment.\n\nNote: This setting will override anything in the `Prompted variables` input with the values from the previous deployment.  In addition, it will use the values from the latest deployment of the Release.",
```

Comment:
I don't think this is going to be able to get the sensitive variable values to copy to the new deployment. It should be called out in the help text here.

Reply:
Ooh, that's a good point, thank you.